src/assisted_installer_controller/assisted_installer_controller.go

```diff
@@ -410,6 +413,13 @@ func (c controller) postInstallConfigs(ctx context.Context) error {
 		return errors.Errorf("Timeout while waiting for console to become available")
 	}
 
+	// Aplly post install manifests:
+	err = utils.WaitForPredicateWithContext(ctx, retryPostManifestTimeout, GeneralWaitInterval, c.applyPostInstallManifests)
+	if err != nil {
+		c.log.WithError(err).Warnf("Failed to apply post manifests.")
+		return err
```

Comment:
@tsorya I think it's enough to return the error here, because we [fail](https://github.com/openshift/assisted-service/blob/master/internal/bminventory/inventory.go#L3976) installation in case assisted-installer  `postInstallConfigs` method return error. Why do you think we should move it before console?

Reply:
Assisted-service will set installed from the point controller will send that console operator is fine. 
Controller doesn't set installed by itself. (and error logic should be moved to service too, we have ticket for it somewhere).
If you will return here error we will move cluster to degraded i think but maybe i miss something
@YuviGold you know this logic the best, can you assist please

Reply:
In case you return an error - the controller will return an error and abort the installation (=cluster will move to "error" state).
In case it's a specific operator manifest that failed to be applied - its status should be updated with "failure".
Otherwise, if it's a general malfunction - I think it means that something isn't right with the cluster - and we can abort the installation. @tsorya WDYT?

Reply:
@machacekondra sorry i forgot this logic. we will not set installed after console was available. Totally forgot, so yes your logic is totally right

Reply:
@tsorya @YuviGold so do we agree that current code is fine. Or should we update the monitor operator states even if custom manifests fails?

Reply:
we agree it is fine for this version. Let rework all retry stuff and push it

Reply:
@tsorya  The retry code is there on line 413.

Reply:
@eranco74 it was agreed to fail the installation for now

Reply:
@machacekondra i meant retry on all the errors and not apply one

Reply:
It is. It is retry on all the errors that can ocure in applyManifest method, including downlaoding kubeapi or creating tempdir.